app/com/linkedin/drelephant/util/InfoExtractor.java

```diff
@@ -106,55 +106,52 @@ public static Scheduler getSchedulerInstance(String appId, Properties properties
    * @param data The Hadoop application data
    */
   public static void loadInfo(AppResult result, HadoopApplicationData data) {
-    Properties properties = new Properties();
-    if (data instanceof MapReduceApplicationData) {
-      properties = retrieveMapreduceProperties((MapReduceApplicationData) data);
-    }
-
+    Properties properties = data.getConf();
```

Comment:
After #162, `getConf` should be sufficient for both `MapReduceApplicationData` and `SparkApplicationData`.